### PR TITLE
Inflation Nomination for sotcsa

### PR DESCRIPTION
**Pull request by sotcsa**
#### Summary of Changes

Fixes #

### CODE REVIEW

File: sdk/src/feature_set.rs [hunk 1]
```diff
@@ -48,6 +48,16 @@ pub mod full_inflation {
...
+
+    pub mod sotcsa {
+        pub mod vote {
+            solana_sdk::declare_id!("EgoekfqCYoraFE5ZkiECGQ945Y5rGBXh3n85sQPuR85r");
+        }
+        pub mod enable {
+            solana_sdk::declare_id!("6f8Y2dACzRjM9R9RwiLp9HuAxo43QwtztgHm4BKUGyxU");
+        }
+    }
+
```
Comment (mvines) [inline sdk/src/feature_set.rs:60]:
```suggestion
```

File: sdk/src/feature_set.rs [hunk 2]
```diff
@@ -217,6 +227,9 @@ lazy_static! {
         (track_writable_deescalation::id(), "track account writable deescalation"),
         (spl_token_v2_self_transfer_fix::id(), "spl-token self-transfer fix"),
         /*************** ADD NEW FEATURES HERE ***************/
+        (full_inflation::sotcsa::vote::id(), "community vote allowing sotcsa to enable full inflation"),
+        (full_inflation::sotcsa::enable::id(), "full inflation enabled by sotcsa"),
+
```
Comment (mvines) [inline sdk/src/feature_set.rs:232]:
```suggestion
```

**Comment by mvines** [comment]
Thanks for the PR @sotcsa, I made a couple minor formatting changes to resolve merge conflicts and pacify the code prettier

**Comment by sotcsa** [comment]
> Thanks for the PR @sotcsa, I made a couple minor formatting changes to resolve merge conflicts and pacify the code prettier

Thanks a lot @mvines !
I made my changes manually because of the patch was conflicting.

**Pull request merged**